### DISCUSSION TITLE
Players claims have no protections from other players

### DISCUSSION THREAD

**Discussion by Logos0** [GriefPrevention/GriefPrevention#1505] [Q&A]
My players can access chests in each other's claims in the world Survival where claims mode Survival is enabled.
Nether of them have have trust status.

Here's my config: https://pastebin.com/tiUqNw1p

You can reach me more easily on discord: Logos #1203

**Comment by bigpresh** [comment]
Please [raise an issue](https://github.com/TechFortress/GriefPrevention/issues/new?assignees=&labels=Unverified+Bug&template=bug-report.md&title=) filling in the stuff asked for in the template - in particular versions, etc, any other plugins installed (and if problem persists with no other plugins loaded), and anything that appears in the logs.

Also, please confirm that it's happening on the worlds listed in your config, and isn't an easy mistake like "it's not being protected in a world that GP hasn't been told to do anything for" :)

**Reply by Logos0** [reply]
Here's more information:
Minecraft 1.16.5
Paper-783
Grief Prevention: 16.17.1

This is happening in the world Survival where claims mode Survival is enabled.
(Config in OP)

Plugins: AuctionHouse, AutomaticBroadcast, Autorank, ChestShop, ChestShop-LegacyIds, CoinFlipper*, CoreProtect, DeadChest, DiscordSRV, EpicPluginLib, Essentials, EssentialsChat, EssentialsGeoIP, Factions, FastAsyncWorldEdit (WorldEdit), GriefPrevention, HolographicDisplays, IllegalStack, ImageMaps, LaggRemover, LuckPerms, MoneyFromMobs, MotdChanger, Multiverse-Core, Multiverse-Inventories, Multiverse-NetherPortals, Multiverse-Portals, PerfectVanish*, PlaceholderAPI, PlayerHeads, PlayMoreSounds, ProtocolLib, PvPManager, pvpstats, Referral, SilkSpawners, SkQuery, Skript, Statz, TAB, Vault, ViaVersion, Votifier, VotingPlugin, Vulcan, WorldBorder, WorldGuard

Nothing is appearing in the logs.

**Any help is appreciated, as my players have been able to freely use chests in each others claims for a week.**
